l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -25,25 +25,28 @@ import {L2CanonicalTransaction} from "../common/Messaging.sol";
 /// @dev The Bridgehub contract serves as the primary entry point for L1<->L2 communication,
 /// facilitating interactions between end user and bridges.
 /// It also manages state transition managers, base tokens, and chain registrations.
+/// Bridgehub is also an IL1AssetHandler for the chains themselves, which is used to migrate the chains
+/// between different parents (for example from L1 to Gateway).
 contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, PausableUpgradeable {
     /// @notice the asset id of Eth
     bytes32 internal immutable ETH_TOKEN_ASSET_ID;
 
-    /// @dev The chain id of L1, this contract will be deployed on multiple layers.
+    /// @dev The chain id of L1. This contract can be deployed on multiple layers, but this value is still equal to the
+    // L1 that is at the most base layer.
     uint256 public immutable L1_CHAIN_ID;
 
-    /// @notice all the ether is held by the weth bridge
+    /// @notice all the ether and ERC20 tokens are held by this bridge.
     IL1AssetRouter public sharedBridge;
 
-    /// @notice we store registered stateTransitionManagers
+    /// @notice StateTransitionManagers that are registered and can be used by the children chains.
```

Comment:
It seems to me that "children chains" will give off an impression that it is somehow related to cross chain communications or that chains are somehow children of Bridgehub. neither is true